packages/arb-ts/src/lib/bridge.ts

```diff
@@ -655,20 +654,20 @@ export class Bridge {
     return BridgeHelper.isWhiteListed(
       address,
       whiteListAddress,
-      this.l1Bridge.l1Provider
+      this.l1Provider
     )
   }
 
   public async setGateways(
     tokenAddresses: string[],
     gatewayAddresses: string[]
   ) {
-    const gasPriceBid = await this.l2Bridge.l2Provider.getGasPrice()
+    const gasPriceBid = await this.l2Provider.getGasPrice()
 
     const maxSubmissionPrice = (
       await this.l2Bridge.getTxnSubmissionPrice(
         // 20 per address, 100 as buffer/ estimate for any additional calldata
-        100 + 20 * (tokenAddresses.length + gatewayAddresses.length)
+        300 + 20 * (tokenAddresses.length + gatewayAddresses.length)
```

Comment:
this actually fails w/ the lower value; this is only a convenience method for internal use, so let's just leave it